src/BenchmarkDotNet/Templates/BuildPlots.R

```diff
@@ -103,17 +113,17 @@ for (file in files) {
 
     for (params in unique(df$Params)) {
       paramsDf <- df %>% filter(Params == params)
-
       paramsDensityPlot <- ggplot(paramsDf, aes(x=Measurement_Value, fill=Job_Id)) +
         ggtitle(paste(title, "/", target, "/", params)) +
         xlab(paste("Time,", timeUnit)) +
         geom_density(alpha=.5, bw="SJ")
       printNice(paramsDensityPlot)
-      ggsaveNice(gsub("-measurements.csv", paste0("-", target, "-", params, "-density.png"), file), paramsDensityPlot)
+      paramsValues <- getParamsValues(c(target,params))
```

Comment:
My last concern: I don't like `paramsValues` and `getParamsValues` names. They are misleading because here we have a combination of the target and the params. It will be better to rename it.

Reply:
Hi, @AndreyAkinshin. I thought about naming, **target** and **params**, both are parts of final picture name, maybe `prefix` and `createPrefix`/`combineParts` will be better? Any suggestions?

Reply:
`prefix` looks good to me.

Reply:
Ok, replaced. Thank you.